src/display/api.js

```diff
@@ -187,13 +188,13 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
 
 /**
  * This is the main entry point for loading a PDF and interacting with it.
- * NOTE: If a URL is used to fetch the PDF data a standard XMLHttpRequest(XHR)
- * is used, which means it must follow the same origin rules that any XHR does
- * e.g. No cross domain requests without CORS.
+ * NOTE: If a URL is used to fetch the PDF data a standard XMLHttpRequest (XHR)
```

Comment:
You probably want to mention the Fetch API here, since that's now the default in modern browsers (with XMLHttpRequest used as fallback).